notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>First comment would be about renaming file to follow the structure suggested in the last meeting (e.g. sitcom-797_ics_&nbsp;slewing_analysis_positioning.ipynb). Also, the title of the notebook can be more informative, e.g. "SITCOM-797 Slewing Analysis: Positioning" or something else.</p><p> </p><p>I believe that some more context will be given in the technote, so, I think being concise here is fine.</p><p> </p><p>One more general comment is regarding the notebook structure. I think you can create subsections to separate the different analysis you are doing (diff between positions, diff vs elevation/azimuth, etc). In my opinion, it makes reader's life easier.</p><p> </p><p>One last thing. I think it would be nice to add a few comments on results of each individual analysis, instead of only having a final conclusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks for the useful suggestions.</p><ul><li>Notebook renamed as suggested</li><li>More explicit title</li><li>Create clearer sub-sections</li></ul>